core/src/tile/tile.cpp

```diff
@@ -73,7 +73,7 @@ void Tile::build(StyleContext& _ctx, const Scene& _scene, const TileData& _data,
                     auto* style = _scene.findStyle(rule.getStyleName());
 
                     if (style) {
-                        rule.eval(_ctx);
+                        if (!rule.eval(_ctx)) { continue; }
```

Comment:
This isn't actually the current behavior of tangram-js, the trello card I created about this was somewhat mis-informed. As I understand it, tangram-js does not check whether the "required" rules of order, priority, or width are set to valid values. We might want that to be the future behavior, but it may cause discrepancies right now. cc @bcamper